src/promptflow-core/promptflow/core/_serving/flow_invoker.py

```diff
@@ -59,7 +60,10 @@ def __init__(
         init_kwargs: dict = None,
         **kwargs,
     ):
-        self.logger = kwargs.get("logger", LoggerFactory.get_logger("flowinvoker"))
+        self.logger = kwargs.get(
+            "logger",
+            LoggerFactory.get_logger("flowinvoker",
+                                     verbosity=kwargs.get('log_level') or logging.INFO))
```

Comment:
Is it expected to use `logging.INFO` if `log_level` is not provided? Would it override the log_level that was set when getting the logger instance? Wonder whether `get_pf_logging_level()` should be used here instead.

Reply:
Fixed, switched to using `get_pf_logging_level()`